### PR TITLE
plat-ls: Add support for armv8 platform flavour

### PR DESCRIPTION
**Pull request by pangupta**
Added support for armv8 platform flavour.
 - PLATFORM = ls-ls1012ardb

Signed-off-by: Pankaj Gupta <pankaj.gupta@nxp.com>

### CODE REVIEW

File: core/arch/arm/plat-ls/conf.mk [hunk 1]
```diff
@@ -20,6 +20,10 @@ CFG_BOOT_SYNC_CPU ?= y
 CFG_BOOT_SECONDARY_REQUEST ?= y
 endif
 
+ifeq ($(PLATFORM_FLAVOR),ls1012ardb)
```
Comment (jforissier) [inline core/arch/arm/plat-ls/conf.mk]: Could be written as:
```make
ifneq (,$(filter $(PLATFORM_FLAVOR),ls1012ardb ls1043ardb))
include core/arch/arm/cpu/cortex-armv8-0.mk
endif
```
But, you may keep the two `ifeq`s if you prefer.

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by b49020** [comment]
`Reviewed-by: Sumit Garg <sumit.garg@nxp.com>`

File: core/arch/arm/plat-ls/conf.mk (hunk 1 above)
Comment (pangupta) [reply core/arch/arm/plat-ls/conf.mk]: Let's not change it for now, When more platforms will be planned to get added, I will change it.

**Comment by pangupta** [comment]
Updated the MAINTAINERS.md.

**Comment by jforissier** [comment]
Thanks @pangupta, can you please add the Acked-by: and Reviewed-by: tags?

**Comment by pangupta** [comment]
Added the Acked-by: and Reviewed-by: tags.

**Pull request merged**